.circleci/config.yml

```diff
@@ -184,6 +184,13 @@ commands:
           command: |
             set GENERATOR_PRESET=express-minimal-test-project
             node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir generated-project-express-minimal
+      - run:
+          name: Generate typescript-minimal project
+          environment:
+            GENERATOR_PRESET: 'typescript-minimal-test-project'
+          command: |
+            set GENERATOR_PRESET=typescript-minimal-test-project
+            node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir generated-project-typescript-minimal
```

Comment:
Was is the bottom line on our job completion time? I want to be cognizant on 1. are these template jobs parallelized? 2. are they slowing things down.